client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -73,11 +74,19 @@ const ConnectedSingleSiteWpcom = connectOptions( ( props ) => {
 
 	useRequestSiteChecklistTaskUpdate( siteId, CHECKLIST_KNOWN_TASKS.THEMES_BROWSED );
 
+	useEffect( () => {
+		if ( ! isNewSearchAndFilter ) {
+			return;
+		}
+
+		document.body.classList.add( 'is-section-themes-i4' );
```

Comment:
Is this a hack...? Or will we improve the logic when we roll out the feature?

Reply:
I just copied the code from [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/blank-canvas/index.js#L15) 😅. Alternatively, we can use this [helper](https://github.com/Automattic/wp-calypso/blob/60302b66f8875e066f76d40e8d2b3209ca8c0ad1/client/layout/body-section-css-class.js#L44) which is essentially the same thing.

Reply:
Mind-blown

Okay, looks like it is an existing pattern in Calypso 😄